contracts/deploy/055_curve_amo.js

```diff
@@ -0,0 +1,478 @@
+      args: [cDripper.address],
+    },
+  ]
+};
+
+
+/**
+ * Deploy Convex ETH Strategy
+ */
+const deployConvexETHMetaStrategy = async ({
+  deployWithConfirmation,
+  withConfirmation,
+  ethers,
+  tokenAddress,
+  poolAddress,
+  gaugeAddress,
+  poolId,
+  crvRewards,
+  cHarvester,
+}) => {
+  const assetAddresses = await getAssetAddresses(hre.deployments);
+  const { deployerAddr } = await getNamedAccounts();
+  const sDeployer = await ethers.provider.getSigner(deployerAddr);
+  const cVaultProxy = await ethers.getContract("OETHVaultProxy");
+  const cVault = await ethers.getContractAt("OETHVault", cVaultProxy.address);
+
+  const dConvexEthMetaStrategyProxy = await deployWithConfirmation(
+    "ConvexEthMetaStrategyProxy"
+  );
+  const cConvexEthMetaStrategyProxy = await ethers.getContract(
+    "ConvexEthMetaStrategyProxy"
+  );
+  const dConvexETHMetaStrategy = await deployWithConfirmation(
+    "ConvexEthMetaStrategy"
+  );
+  const cConvexETHMetaStrategy = await ethers.getContractAt(
+    "ConvexEthMetaStrategy",
+    dConvexEthMetaStrategyProxy.address
+  );
+  await withConfirmation(
+    cConvexEthMetaStrategyProxy
+      .connect(sDeployer)
+      ["initialize(address,address,bytes)"](
+        dConvexETHMetaStrategy.address,
+        deployerAddr,
+        []
+      )
+  );
+
+  console.log("Initialized ConvexETHMetaStrategyProxy");
+  const initFunction =
+    "initialize(address[],address[],address[],(address,address,address,address,address,address,address,uint256))";
+
+  await withConfirmation(
+    cConvexETHMetaStrategy.connect(sDeployer)[initFunction](
+      [assetAddresses.CVX, assetAddresses.CRV],
+      [addresses.mainnet.WETH],
+      [tokenAddress],
+      [
+        poolAddress,
+        cVaultProxy.address,
+        addresses.mainnet.CVXBooster,
+        addresses.mainnet.OETHProxy,
+        addresses.mainnet.WETH,
+        crvRewards,
+        tokenAddress,
+        poolId, // TODO change this fork poolId with the mainnet one!
+      ]
+    )
+  );
+  console.log("Initialized ConvexETHMetaStrategy");
+  await withConfirmation(
+    cConvexETHMetaStrategy.connect(sDeployer).transferGovernance(guardianAddr)
+  );
+  console.log(
+    `ConvexETHMetaStrategy transferGovernance(${guardianAddr} called`
+  );
+
+  return [
+    {
+      // Claim Vault governance
+      contract: cConvexETHMetaStrategy,
+      signature: "claimGovernance()",
+      args: [],
+    },
+    {
+      // Claim Vault governance
+      contract: cConvexETHMetaStrategy,
+      signature: "setHarvesterAddress(address)",
+      args: [cHarvester.address],
+    },
+    {
+      contract: cHarvester,
+      signature: "setSupportedStrategy(address,bool)",
+      args: [cConvexETHMetaStrategy.address, true],
+    },
+    // Set reward token config
+    {
+      contract: cHarvester,
+      // tokenAddress, allowedSlippageBps, harvestRewardBps, uniswapV2CompatibleAddr, liquidationLimit, doSwapRewardToken
```

Comment:
This comment could move to the args below?

Reply:
I has already been deployed as part of [this deploy](https://github.com/OriginProtocol/origin-dollar/pull/1334) but not configured.

Reply:
Yes, but then it would be less confusing 🙃 

... moved